server/rosetta/client_online.go

```diff
@@ -85,14 +61,74 @@ func NewClient(cfg *Config) (*Client, error) {
 		v = "unknown"
 	}
 
+	txConfig := authtx.NewTxConfig(cfg.Codec, authtx.DefaultSignModes)
+
+	var supportedOperations []string
+	for _, ii := range cfg.InterfaceRegistry.ListImplementations("cosmos.base.v1beta1.Msg") {
+		resolvedMsg, err := cfg.InterfaceRegistry.Resolve(ii)
+		if err != nil {
+			continue
+		}
+
+		if _, ok := resolvedMsg.(sdk.Msg); ok {
+			supportedOperations = append(supportedOperations, strings.TrimLeft(ii, "/"))
+		}
+	}
+
+	supportedOperations = append(
+		supportedOperations,
+		bank.EventTypeCoinSpent, bank.EventTypeCoinReceived,
+	)
+
 	return &Client{
-		config:  cfg,
-		ir:      cfg.InterfaceRegistry,
-		version: fmt.Sprintf("%s/%s", info.AppName, v),
+		supportedOperations: supportedOperations,
+		config:              cfg,
+		auth:                nil,
+		bank:                nil,
+		tmRPC:               nil,
+		version:             fmt.Sprintf("%s/%s", info.AppName, v),
+		converter:           NewConverter(cfg.Codec, cfg.InterfaceRegistry, txConfig),
 	}, nil
 }
 
-func (c *Client) accountInfo(ctx context.Context, addr string, height *int64) (auth.AccountI, error) {
+// Bootstrap is gonna connect the client to the endpoints
+func (c *Client) Bootstrap() error {
```

Comment:
Let's rename this method to `Dial` or `Connect`.

Reply:
this is part of interface implementation